util/gpool/spmc/option.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spmc
+
+import (
+	"time"
+)
+
+// Option represents the optional function.
+type Option func(opts *Options)
+
+func loadOptions(options ...Option) *Options {
+	opts := new(Options)
+	for _, option := range options {
+		option(opts)
+	}
+	if opts.LimitDuration == 0 {
+		opts.LimitDuration = 200 * time.Millisecond
+	}
+	return opts
+}
+
+// Options contains all options which will be applied when instantiating an pool.
+type Options struct {
+	// PanicHandler is used to handle panics from each worker goroutine.
+	// if nil, panics will be thrown out again from worker goroutines.
+	PanicHandler func(interface{})
+
+	// ExpiryDuration is a period for the scavenger goroutine to clean up those expired workers,
+	// the scavenger scans all workers every `ExpiryDuration` and clean up those workers that haven't been
+	// used for more than `ExpiryDuration`.
+	ExpiryDuration time.Duration
+
+	// LimitDuration is a period in the limit mode.
+	LimitDuration time.Duration
```

Comment:
It seems that we don't use `LimitDuration` and `Nonblocking`, are there be used in the next PR？If there are not, please remove them.